Governance/manage/manage-intro.md

```diff
@@ -0,0 +1,30 @@
+# Guidelines to manage climate dataset
+
+These guidelines are aimed to dataset managers and cover all aspects of managing an authoritative or a replicated dataset on a local server.
```

Comment:
These guidelines are aimed to address

Reply:
Or "aimed to assist dataset managers"?
(nomenclature: managers? curators? stewards? ping @chloemackallah )